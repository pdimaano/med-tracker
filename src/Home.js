import React from "react";
import { Card, CardBody, CardTitle, CardText } from "reactstrap";

/** Home Page
 *
 *  Props: none
 */

function Home() {
  return (
    <section className="col-md-8">
      <Card>
        <CardBody className="text-center">
          <CardTitle>
            <p className="fw-bold" style={{ fontSize: "1.5em" }}>
              Daily Medication Tracker
            </p>
          </CardTitle>
          <CardText>
            For every day of the week
          </CardText>
        </CardBody>
      </Card>
    </section>
  );
}

export default Home;
