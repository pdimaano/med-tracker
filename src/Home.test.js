import React from "react";
import { render } from "@testing-library/react";
import Home from "./Home";

describe("Home component", function () {
  it("renders the home component", function () {
    render(<Home />);
  });

  it("matches a snapshot", function () {
    const { getByText, asFragment } = render(
      <Home />
    );
    getByText(/Daily Medication/);

    expect(asFragment()).toMatchSnapshot();
  })
})