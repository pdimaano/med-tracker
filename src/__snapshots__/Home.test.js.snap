// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component matches a snapshot 1`] = `
<DocumentFragment>
  <section
    class="col-md-8"
  >
    <div
      class="home-card card"
    >
      <div
        class="text-center card-body"
      >
        <div
          class="card-title"
        >
          <p
            class="fw-bold"
            style="font-size: 1.5em;"
          >
            Daily Medication Tracker
          </p>
        </div>
        <p
          class="card-text"
        >
          Never forget your daily meds again!
        </p>
        <img
          alt="Rx symbol"
          class="rx-image"
          src="Rx_symbol.png"
        />
      </div>
    </div>
  </section>
</DocumentFragment>
`;
